### DISCUSSION TITLE
Fold all and Unfold all. Do we have this features already?

### DISCUSSION THREAD

**Discussion by Ilham-Habibullin** [zed-industries/zed#8508] [Feature Requests]
It seems like this is not implemented yet, there is only `fold/unfold single block` and `fold selected ranges`. 
Or if this is already implemented can you explain how to use that? 
Because it is not searchable among commands in `cmd+shift+p`. 
Thank you!

**Comment by Moshyfawn** [comment]
This was discussed in #8015.

As you mentioned, single block folding is already available via

```json
"bindings": {
  "alt-cmd-[": "editor::Fold",
  "alt-cmd-]": "editor::UnfoldLines"
}

```

Folding at a specific level _(and all levels)_ is currently a feature request in #5142.

**Reply by Ilham-Habibullin** [reply]
thanks!

**Reply by Moshyfawn** [reply]
Np!
